resources/guidelines/testing/store/quality-guidelines-plugins/README.md

```diff
@@ -244,6 +246,35 @@ A test button for optional API access data must be available. If the API data is
 
 [Example for implementing an API Test Button into the System Config form](https://github.com/shyim/ShyimApiTest)
 
+### External technology apps / STP apps
+
+Every external technology app needs to track their commission. This is an example to implement the tracking 
+
+    {
+      "identifier": "8e167662-6bbb-11eb-9439-0242ac130002",
+      "reportDate": "2005-08-15T15:52:01",
+      "instanceId": "alur24esfaw3ghk",
+      "shopwareVersion": "6.3.1",
+      "reportDataKeys": [
+        {
+          "customer": 3
+        },
+        {
+          "turnover": 440
+        }
+      ]
+    }
+
+identifier*: uuid                      Identifier used to validate the request
+
+reportDate*: date-time                 Date from when the given information are. Needs to be in DATE_ATOM format
+
+instanceId: string                     The id used to identify a shop can be send, when the given data is provided by a shop
+
+shopwareVersion: string                The shopware version of a shop, can be send, when the given data is provided by a shop
+
+reportDataKeys*:                       Contains the information, which are being tracked 
```

Comment:
Since I have no further information on the subject, for example, which request is meant, we can remove the information on the individual points.

The manufacturers will receive a separate information sheet from the alliance team about the information.